src/test/scala/io/iohk/ethereum/ledger/BlockRewardSpec.scala

```diff
@@ -0,0 +1,71 @@
+    afterRewardWorldState.getGuaranteedAccount(Address(block.header.beneficiary)).balance shouldEqual (beforeExecutionBalance + minerTwoOmmersReward)
+  }
+
+  "Reward" should "be paid to the miner even if the account doesn't exist" in new TestSetup {
+    val block = sampleBlock(Address(0xdeadbeef))
+    val afterRewardWorldState: InMemoryWorldStateProxy = Ledger.payBlockReward(Config.Blockchain.BlockReward, block, worldState)
+    afterRewardWorldState.getGuaranteedAccount(Address(block.header.beneficiary)).balance shouldEqual Config.Blockchain.BlockReward
+  }
+
+  "Reward Calculation" should "be paid if ommers are included in block" in new TestSetup {
+    val block = sampleBlock(validAccountAddress, Seq(validAccountAddress2, validAccountAddress3))
+    val afterRewardWorldState: InMemoryWorldStateProxy = Ledger.payBlockReward(Config.Blockchain.BlockReward, block, worldState)
+    val beforeExecutionBalance1: BigInt = worldState.getGuaranteedAccount(Address(block.header.beneficiary)).balance
+    val beforeExecutionBalance2: BigInt = worldState.getGuaranteedAccount(Address(block.body.uncleNodesList.head.beneficiary)).balance
+    val beforeExecutionBalance3: BigInt = worldState.getGuaranteedAccount(Address(block.body.uncleNodesList(1).beneficiary)).balance
+    afterRewardWorldState.getGuaranteedAccount(Address(block.header.beneficiary)).balance shouldEqual (beforeExecutionBalance1 + minerTwoOmmersReward)
+    afterRewardWorldState.getGuaranteedAccount(Address(block.body.uncleNodesList.head.beneficiary)).balance shouldEqual (beforeExecutionBalance2 + ommerFiveBlocksDifferenceReward)
+    afterRewardWorldState.getGuaranteedAccount(Address(block.body.uncleNodesList(1).beneficiary)).balance shouldEqual (beforeExecutionBalance3 + ommerFiveBlocksDifferenceReward)
+  }
+
+  "Reward" should "be paid if ommers are included in block even if accounts don't exist" in new TestSetup {
+    val block = sampleBlock(Address(0xdeadbeef), Seq(Address(0x1111), Address(0x2222)))
+    val afterRewardWorldState: InMemoryWorldStateProxy = Ledger.payBlockReward(Config.Blockchain.BlockReward, block, worldState)
+    afterRewardWorldState.getGuaranteedAccount(Address(block.header.beneficiary)).balance shouldEqual minerTwoOmmersReward
+    afterRewardWorldState.getGuaranteedAccount(Address(block.body.uncleNodesList.head.beneficiary)).balance shouldEqual ommerFiveBlocksDifferenceReward
+    afterRewardWorldState.getGuaranteedAccount(Address(block.body.uncleNodesList(1).beneficiary)).balance shouldEqual ommerFiveBlocksDifferenceReward
+  }
+
+
+  trait TestSetup {
+    val storagesInstance = new SharedEphemDataSources with Storages.DefaultStorages
+
+    val validAccountAddress = Address(0xababab)
+    val validAccountAddress2 = Address(0xcdcdcd)
+    val validAccountAddress3 = Address(0xefefef)
+
+    val minerTwoOmmersReward = BigInt("5312500000000000000")
+    val ommerFiveBlocksDifferenceReward = BigInt("1875000000000000000")
```

Comment:
minor - maybe calculated values so it is easier to change and easier to understand?